src/tools/manifest-checker.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import minimist from 'minimist';
+import {Manifest, ManifestWarning} from '../runtime/manifest.js';
+import {PlatformLoader} from '../platform/loader-web.js';
+
+// Script to check that a bundle of Arcs manifest files, particle
+// implementations and JSON data files is complete (i.e. no explicitly mentioned
+// dependencies are missing).
+//
+// To be run with the arcs_manifest BUILD rule.
+
+/**
+ * Loads the given .arcs manifest file and checks it for errors. Errors are
+ * thrown as an exception.
+ */
+async function checkManifest(src: string) {
+  const loader = new PlatformLoader({});
+  const manifest = await Manifest.load(src, loader);
```

Comment:
I'd verify that relative import statements are working. In the past we ran into some issues (especially in g3) with the loader not handling relative imports. They seem to have gone away, but I'd at least try it manually locally to make sure.

Reply:
Yep they work. I was testing the file `particles/Native/Wasm/example.arcs`, and played around with all the interesting cases, like relative imports, stores, etc.

I considered writing unit tests for this manifest-checker.ts file, but ultimately decided it wasn't worth it. I'd have to override too much of the Loader class to get the unit tests to work in a fake environment, and at that point we aren't really testing the implementation anymore.